packages/ui/src/council/modals/VoteForCouncil/VoteForCouncilModal.tsx

```diff
@@ -29,12 +28,9 @@ export const VoteForCouncilModal = () => {
 
   const constants = useCouncilConstants()
   const minStake = constants?.election.minVoteStake
-  const requiredStake = minStake?.toNumber() ?? 0
+  const requiredStake = minStake as BN
```

Comment:
it won't be necessary when this will be merged https://github.com/Joystream/pioneer/pull/3265